mod/phpmailer/start.php

```diff
@@ -226,37 +210,34 @@ function phpmailer_send($to, $to_name, $subject, $body, array $bcc = NULL, $html
 
 	$is_ssl    = elgg_get_plugin_setting('ep_phpmailer_ssl', 'phpmailer');
 	$ssl_port  = elgg_get_plugin_setting('ep_phpmailer_port', 'phpmailer');
-	try{
-	if ($is_smtp && isset($smtp_host)) {
-		//$phpmailer->IsSMTP();
-		//$phpmailer->Host = $smtp_host;
-		//$phpmailer->SMTPAuth = false;
-		if ($smtp_auth) {
-			//$phpmailer->SMTPAuth = true;
-			//$phpmailer->Username = elgg_get_plugin_setting('phpmailer_username', 'phpmailer');
-			//$phpmailer->Password = elgg_get_plugin_setting('phpmailer_password', 'phpmailer');
-
-			if ($is_ssl) {
+	
+	try {
+
+		if ($is_smtp && isset($smtp_host)) {
+
+			if ($smtp_auth && $is_ssl) {
+
 				$phpmailer->SMTPSecure = "tls";
 				$phpmailer->Port = $ssl_port;
+				
 			}
+		} else {
+			// use php's mail
+			$phpmailer->IsMail();
 		}
-	}
-	else {
-		// use php's mail
-		$phpmailer->IsMail();
-	}
-	//error_log("Sending email from => " . $phpmailer->FromName );
-	$return = $phpmailer->Send();
-	//error_log("Sent email from => " . $phpmailer->FromName );
-	} catch(phpmailerException $e) {
+	
+		$return = $phpmailer->Send();
+
+	} catch (phpmailerException $e) {
 		error_log($e->errorMessage());
 	}
 	
 
 	if (!$return ) {
 		error_log('PHPMailer error: ' . $phpmailer->ErrorInfo, 'WARNING');
-	} /* else
-		error_log("email sent"); */
+	} else
+		error_log("email sent");
```

Comment:
see the description of the pull request (there is an issue for it, however, that was tied to a different pull request that was merged and closed off). I have no way to debug other than to code in more logging code.